willa/web/app.py

```diff
@@ -117,6 +157,7 @@ async def chat(message: cl.Message) -> None:
 
         if 'ai_message' in reply:
             await cl.Message(content=reply['ai_message']).send()
+            cl.context.current_run.metadata['langfuse_trace_id'] = reply['langfuse_trace_id']
```

Comment:
Save the trace ID into the run metadata, which is where Chainlit stores Feedback objects (forId -> the run, not the message).